### PR TITLE
Display error in WasmExecutionError::ExecuteModuleInWasmer

### PR DESCRIPTION
**Pull request by christos-h**
## Motivation

Wasmer errors were not being displayed resulting in making debugging applications difficult.

## Proposal

Display Wasmer error in the error message.

## Test Plan

Has been tested manually.

### CODE REVIEW

File: linera-execution/src/wasm/mod.rs [hunk 1]
```diff
@@ -223,7 +223,7 @@ pub enum WasmExecutionError {
     CreateWasmtimeEngine(#[source] anyhow::Error),
     #[cfg(with_wasmer)]
     #[error(
-        "Failed to execute Wasm module in Wasmer. This may be caused by panics or insufficient fuel."
+        "Failed to execute Wasm module in Wasmer. This may be caused by panics or insufficient fuel. {0}"
```
Comment (ma2bd) [inline linera-execution/src/wasm/mod.rs:226]: In my experience, the error was always the same here (hence the vague explanation). Perhaps something to test again after wasmer is updated?

**Review by jvff — CHANGES_REQUESTED** [review]
The `#[from]` attribute adds the `::wasmer::RuntimeError` as the source. This should get displayed when the error is printed. If it is not, maybe we should fix how it is displayed instead of adding redundant information to the error message?

File: linera-execution/src/wasm/mod.rs (hunk 1 above)
Comment (christos-h) [reply linera-execution/src/wasm/mod.rs:226]: I think this helps at the GraphQL level - the inner error wasn't being rendered in the response otherwise.

**Review by jvff — APPROVED** [review]
Approved! As we discussed on call, we already have this same change for Wasmtime, so we can apply this change as is for Wasmer and open an issue to more generically add the error sources to the error string reported by GraphQL.

**Pull request merged**